regression-tests/run-tests.sh

```diff
@@ -195,7 +195,7 @@ for test_file in $tests; do
         # The source is temporarily copied to avoid issues with bash paths in cl.exe
         (cd $exec_out_dir; \
          cp ../../$expected_src $generated_cpp_name;
-         $compiler_cmd"$test_bin" \
+         $compiler_cmd "$test_bin" \
```

Comment:
Does it work with the space? There was a point where it was necessary to glue the two together. Maybe some cleaning I did recently got rid of this requirement...

Reply:
Maybe it's because I added `continue-on-error: true` (<https://github.com/hsutter/cppfront/pull/934#discussion_r1450753377>).